source/extensions/filters/network/http_connection_manager/dependency_manager.cc

```diff
@@ -0,0 +1,31 @@
+#include "extensions/filters/network/http_connection_manager/dependency_manager.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace HttpConnectionManager {
+
+using envoy::extensions::filters::common::dependency::v3::Dependency;
+
+bool DependencyManager::decodePathIsValid() {
+  auto cmp = [](Dependency a, Dependency b) { return a.name() != b.name(); };
+  std::set<Dependency, decltype(cmp)> satisfied(cmp);
```

Comment:
While I'd also prefer the absl library, it seems not to support the custom comparator API in the same way, and I'm going to avoid the hassle of figuring that out unless you feel strongly.